### ISSUE TITLE
please fix for 24h2

### ISSUE THREAD

**Issue by Pezhmankiani** [hzqst/VmwareHardenedLoader#65]
Hi
this software not working on windows 11 24h2
please release an update to fix this issue.
work on 23h2 without problem

**Comment by W4ZM** [comment]
i agree, it cant load the driver in 24h2 version , now i need to go back to win 10, what a hustle ...

**Comment by itsOwen** [comment]
> i agree, it cant load the driver in 24h2 version , now i need to go back to win 10, what a hustle ...

I am working on a solution might update it

**Comment by Pezhmankiani** [comment]
Hello,

Thank you for your great work on VMHardened Loader.

I’m currently testing it with Windows 11 Pro (24H2 build 22631) running inside VMware Fusion on a real Intel-based Mac. Everything works as expected in most cases, but I’m facing an issue with one specific software:

**FLEX by MagicMotorsport (https://magicmotorsport.com/flex/)**

Despite using the loader and all available stealth configurations (SMBIOS spoofing, CPUID masking, backdoor isolation, etc.), the FLEX application still detects the virtualized environment and refuses to run.

I’ve already disabled Paravirtualization, applied full BIOS/SMBIOS overrides, used smbios.reflectHost, and even attempted to use a patched `bios.rom`. However, the detection persists—likely via ACPI tables or direct CPUID instructions.

Could you please confirm if there’s any plan to update VMHardened Loader for Windows 11 24H2 compatibility and specifically address detection vectors used by FLEX?

Thank you for your time and support.

**Comment by itsOwen** [comment]
> Hello,
> 
> Thank you for your great work on VMHardened Loader.
> 
> I’m currently testing it with Windows 11 Pro (24H2 build 22631) running inside VMware Fusion on a real Intel-based Mac. Everything works as expected in most cases, but I’m facing an issue with one specific software:
> 
> **FLEX by MagicMotorsport (https://magicmotorsport.com/flex/)**
> 
> Despite using the loader and all available stealth configurations (SMBIOS spoofing, CPUID masking, backdoor isolation, etc.), the FLEX application still detects the virtualized environment and refuses to run.
> 
> I’ve already disabled Paravirtualization, applied full BIOS/SMBIOS overrides, used smbios.reflectHost, and even attempted to use a patched `bios.rom`. However, the detection persists—likely via ACPI tables or direct CPUID instructions.
> 
> Could you please confirm if there’s any plan to update VMHardened Loader for Windows 11 24H2 compatibility and specifically address detection vectors used by FLEX?
> 
> Thank you for your time and support.

I am not the developer of VMhwardendloader, I was just working on a fix for 11, If i get some free time i might continue working on it or even share it

**Comment by curious-youna** [comment]
> I am not the developer of VMhwardendloader, I was just working on a fix for 11, If i get some free time i might continue working on it or even share it

Please update us :)

**Comment by chris6611** [comment]
> > Hello,
> > Thank you for your great work on VMHardened Loader.
> > I’m currently testing it with Windows 11 Pro (24H2 build 22631) running inside VMware Fusion on a real Intel-based Mac. Everything works as expected in most cases, but I’m facing an issue with one specific software:
> > **FLEX by MagicMotorsport (https://magicmotorsport.com/flex/)**
> > Despite using the loader and all available stealth configurations (SMBIOS spoofing, CPUID masking, backdoor isolation, etc.), the FLEX application still detects the virtualized environment and refuses to run.
> > I’ve already disabled Paravirtualization, applied full BIOS/SMBIOS overrides, used smbios.reflectHost, and even attempted to use a patched `bios.rom`. However, the detection persists—likely via ACPI tables or direct CPUID instructions.
> > Could you please confirm if there’s any plan to update VMHardened Loader for Windows 11 24H2 compatibility and specifically address detection vectors used by FLEX?
> > Thank you for your time and support.
> 
> I am not the developer of VMhwardendloader, I was just working on a fix for 11, If i get some free time i might continue working on it or even share it

Hey mate, any update on your fix progress?

**Comment by estudiovertigo-code** [comment]
I managed to get Windows 11 up to 23h2. works like a charm

**Comment by hzqst** [comment]
have fixed in the latest build.
